content/developer/howtos/translations.rst

```diff
@@ -156,14 +156,14 @@ Plural
 
 **Don't** pluralize terms the English-way::
 
-    msg = _("You have %(count)s invoice", count=invoice_count)
+    msg = _("You have %(count) invoice", count=invoice_count)
     if invoice_count > 1:
       msg += _("s")
 
 **Do** keep in mind every language has different plural forms::
 
     if invoice_count > 1:
-      msg = _("You have %(count)s invoices", count=invoice_count)
+      msg = _("You have %(count) invoices", count=invoice_count)
```

Comment:
The format for labeled placeholders is `%(placeholder)s`. This change is not correct.